[2024-12-20T10:51:28.851+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-20T10:51:28.884+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_operator_dag_v3.SQL_delete_dag_if_exists scheduled__2024-01-01T04:00:00+00:00 [queued]>
[2024-12-20T10:51:28.904+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_operator_dag_v3.SQL_delete_dag_if_exists scheduled__2024-01-01T04:00:00+00:00 [queued]>
[2024-12-20T10:51:28.906+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-20T10:51:28.931+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): SQL_delete_dag_if_exists> on 2024-01-01 04:00:00+00:00
[2024-12-20T10:51:28.950+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=16870) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-20T10:51:28.953+0000] {standard_task_runner.py:72} INFO - Started process 16878 to run task
[2024-12-20T10:51:28.967+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sql_operator_dag_v3', 'SQL_delete_dag_if_exists', 'scheduled__2024-01-01T04:00:00+00:00', '--job-id', '5508', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgresql.py', '--cfg-path', '/tmp/tmp2ylqd0sa']
[2024-12-20T10:51:28.972+0000] {standard_task_runner.py:105} INFO - Job 5508: Subtask SQL_delete_dag_if_exists
[2024-12-20T10:51:29.083+0000] {task_command.py:467} INFO - Running <TaskInstance: sql_operator_dag_v3.SQL_delete_dag_if_exists scheduled__2024-01-01T04:00:00+00:00 [running]> on host a998b5b3d359
[2024-12-20T10:51:29.265+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mateo' AIRFLOW_CTX_DAG_ID='sql_operator_dag_v3' AIRFLOW_CTX_TASK_ID='SQL_delete_dag_if_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T04:00:00+00:00'
[2024-12-20T10:51:29.270+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-20T10:51:29.315+0000] {sql.py:278} INFO - Executing: 
            delete from dag_runs_v2 where dt='2024-01-01' and dag_id = 'sql_operator_dag_v3';
        
[2024-12-20T10:51:29.347+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:51:29.365+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:51:29.386+0000] {sql.py:553} INFO - Running statement: 
            delete from dag_runs_v2 where dt='2024-01-01' and dag_id = 'sql_operator_dag_v3';
        , parameters: None
[2024-12-20T10:51:29.391+0000] {sql.py:562} INFO - Rows affected: 0
[2024-12-20T10:51:29.425+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-20T10:51:29.427+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sql_operator_dag_v3, task_id=SQL_delete_dag_if_exists, run_id=scheduled__2024-01-01T04:00:00+00:00, execution_date=20240101T040000, start_date=20241220T105128, end_date=20241220T105129
[2024-12-20T10:51:29.688+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-20T10:51:29.791+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-20T10:51:29.801+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
